colrev/ops/built_in/prescreen/scope_prescreen.py

```diff
@@ -218,9 +233,22 @@ def run_prescreen(
     ) -> dict:
         """Prescreen records based on the scope parameters"""
 
+        repeat_input_question = True
+        include_unranked_journals = False
```

Comment:
The `include_unranked_journals` variable is no longer necessary due to changes in `main` ([1](https://github.com/CoLRev-Environment/colrev/blob/9338defabd04b0997b62b92769e3b1a7a5e2215c/colrev/ops/built_in/prescreen/scope_prescreen.py#L215), [2](https://github.com/CoLRev-Environment/colrev/blob/main/colrev/ops/built_in/prescreen/scope_prescreen.md)).